apps/site-landing/src/containers/ReturnProfile/data/params.ts

```diff
@@ -35,7 +35,7 @@ type ShockAbsorber = {
 // The data we use to calculate a clients potential returns
 export type SimulationParameters = {
   // How many months should we be calculating this for?
-  maxDuration: DurationObject;
+  //maxDuration: DurationObject;
```

Comment:
Can this be removed?

Reply:
yeah, of course.  It seems pretty unlikely we'll go back to the old way